core/src/main/kotlin/transformers/descriptors/DefaultDescriptorToDocumentationTranslator.kt

```diff
@@ -58,7 +60,16 @@ class DokkaDescriptorVisitor(
         val dri = parent.withClass(descriptor.name.asString())
         val scope = descriptor.getMemberScope(emptyList())
         val descriptorData = descriptor.takeUnless { it.isExpect }?.resolveClassDescriptionData()
-        return Class(
+        val enum_values =
+            if (descriptor.kind == org.jetbrains.kotlin.descriptors.ClassKind.ENUM_CLASS) {
+                (descriptor as LazyClassDescriptor).unsubstitutedMemberScope.getVariableNames()
+                    .map { it.asString() }
+                    .filter { !listOf("name", "ordinal").contains(it) }
```

Comment:
Won't work if enum has any properties.

Reply:
You can just take children classes (they are already extracted in this method) and then filter them using their `kind`.

Reply:
Changed.